ntp.go

```diff
@@ -38,6 +38,15 @@ func (t ntpTime) UTC() time.Time {
 	return time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC).Add(time.Duration(nsec))
 }
 
+func toNtpTime(t time.Time) ntpTime {
+	seconds := uint32(t.Unix() - time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC).Unix())
+	fraction := t.Nanosecond() << 32
```

Comment:
You're right. The time library converts their internal int 64 into a platform int. I will turn it back into int64 before shifting.